### PR TITLE
[CI:DOCS] fix misleading save/load usage

### PR DESCRIPTION
**Pull request by QiWang19**
Fix the container archive description in podman save/load docs that may lead to misusing the save/load instead of import/export for containers.
related https://github.com/containers/podman/issues/8407#issuecomment-732414865
Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL

**Review by Luap99 — COMMENTED** [review]
LGTM

File: cmd/podman/images/load.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ var (
 	loadDescription = "Loads an image from a locally stored archive (tar file) into container storage."
 	loadCommand     = &cobra.Command{
 		Use:               "load [options] [NAME[:TAG]]",
-		Short:             "Load an image from container archive",
+		Short:             "Load image(s) from tar archive",
```
Comment (TomSweeneyRedHat) [inline cmd/podman/images/load.go:26]:
```suggestion
		Short:             "Load image(s) from a tar archive",
```

File: docs/source/markdown/podman-load.1.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 % podman-load(1)
 
 ## NAME
-podman\-load - Load an image from a container image archive into container storage
+podman\-load - Load image(s) from tar archive into container storage
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-load.1.md:4]:
```suggestion
podman\-load - Load image(s) from a tar archive into container storage
```

**Comment by TomSweeneyRedHat** [comment]
A couple of small suggestions, it's not wrong as is, but I think it's better with the 'a'.

**Review by vrothberg — COMMENTED** [review]
I would to love save/load to explain the difference to export/import and the vice versa. It's a very frequent trap and giving users more information may help them find what they're looking for.

**Comment by QiWang19** [comment]
@containers/podman-maintainers PTAL

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**